TestProject1/ProFormaParserTests.cs

```diff
@@ -97,6 +97,17 @@ public void MixedDescriptor()
             Assert.AreEqual("+14.02", tag.Descriptors.Last().Value);
         }
 
+        [Test]
+        [TestCase("PROTEOFXRM")]
+        [TestCase("PROTEOF@RM")]
+        [TestCase("proteoform")]
```

Comment:
We could add one with a '*' for a stop codon to make sure we don't break that in the future somehow.